src/factories.py

```diff
@@ -84,6 +84,22 @@ class Meta:
     name = factory.Sequence(lambda n: f'Task {n}')
     created_by = factory.SubFactory(UserFactory)
 
+    @factory.post_generation
+    def solutions(self, create, extracted, **kwargs):
+        if not create:
+            return
+
+        if extracted:
```

Comment:
slight improvement to this factory could be like we do on `SubissionScoreFactory` and check to see if `isinstance(extracted, Solution)` and then `self.solutions.add(extracted)` else do the `for solution in extracted` so that we can pass either a single solution object, _or_ a list of them to this factory.

Reply:
I think we should have a convention where we just always pass an array to this? hmm... 

that way we don't have to modify _every_ factory, we just know for m2m/whatever we pass `[single_item]`

Reply:
🤷‍♂ yea, phases is that way, always have to pass a list of tasks. Does seem weird to always have to remember to wrap it in a list, since most of the time w/ a factory, we are probably only going to be passing one element anyway...

Reply:
Yeah it is kind of weird, it's the same way with sending emails `send_mail(from=one_email, to=[list_of_emails])` even though I'm usually just sending an email to one person, that's how the interface works =\

I'll think about this a bit more.. hmm...

Reply:
Also, in this case we're passing to a "plural variable:"

```
task = TaskFactory(created_by=user, solutions=[solution])

...

phase = PhaseFactory(competition=competition, tasks=[task])
```

I _think_ this pattern makes sense..